gateway/db/init.sql

```diff
@@ -1,2 +1,7 @@
+-- Copyright (c) 2023, Cisco and/or its affiliates.
+-- All rights reserved.
+-- See LICENSE file in this distribution.
+-- SPDX-License-Identifier: Apache-2.0
+
 CREATE DATABASE tiedie;
 
```

Comment:
have you tested this?

Reply:
Yes, I tested it. That is a valid SQL comment.